cves/CVE-2017-14056.yml

```diff
@@ -64,7 +64,21 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  In order to decode a RL2 file, a video codec for the 1993 adult video game 
+  "Vouyer", the system must first pass it through a demuxer, a process that 
+  separates the audio and video data into seperate streams of information that 
+  can be easily used by the rest of the system. The RL2 file has a header that 
+  specifies the amount of frames the video will have. It then uses this value
+  to go through each frame and get three values, the size of both the video and 
+  audio data, how far away the frame is from the start of the file, and the size 
+  of just the audio data. However, there is no check for an end of file character 
+  in these loops, which could lead to a potential Denial of Service attack. An 
+  attacker could modify the number of frames in the header to a very large 
+  number, which could cause the program to loop through this process much more 
+  than intended. Since the program needs to save the chunk and audio frame data,
+  a bunch of garbage data could fill up the systems memory, and put a large strain
+  on the systems CPU. This could make either the machine or software unusable. 
```

Comment:
Very informative and detailed. You do a good job of explaining how a Denial of Service attack could occur and how bad data could cause the system to become unstable.